config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -27,7 +27,8 @@ spec:
         apiVersion:
           description: 'APIVersion defines the versioned schema of this representation
             of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            internal value, and may reject unrecognized values. 
+            More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
Those are generated automatically by running hack/update-crd.sh so we are not supposed to modify them manually.

Reply:
Reverted URL update for CRDs.